tools/cli/go-whisk-cli/commands/property.go

```diff
@@ -190,8 +190,7 @@ var propertyUnsetCmd = &cobra.Command{
                     map[string]interface{}{"ok": color.GreenString("ok:")}))
             if len(DefaultAuth) > 0 {
                 okMsg += fmt.Sprintf(
-                    wski18n.T("; the default value of {{.default}} will be used.\n",
-                        map[string]interface{}{"default": boldString(DefaultAuth)}))
+                    wski18n.T("; the default value will be used. Run 'wsk property get --auth' to see the new value.\n"))
```

Comment:
should the error message remove the reference to "default value" here since there is no default value or rather the default value is undefined?